tedana/decomposition/pca.py

```diff
@@ -148,19 +147,16 @@ def tedpca(data_cat, data_oc, combmode, mask, adaptive_mask, t2sG,
     Outputs:
 
     This function writes out several files:
-
-    ===========================    =============================================
-    Filename                       Content
-    ===========================    =============================================
-    desc-PCA_decomposition.json    PCA component table
-    desc-PCA_mixing.tsv            PCA mixing matrix
-    desc-PCA_components.nii.gz     Component weight maps
-    ===========================    =============================================
+    - PCA component table
+    - PCA mixing matrix
+    - z-scored PCA components
```

Comment:
This PR removes the file names from this description. I'd lean towards keeping those file names in  and maybe change the header to "Default Filename"
The benefit of removing the file name is, if we keep them, that means, every time we change file names, we'll also need to change the defaults in each of these help texts, but I think the clarity benefits of including the file names is still more useful.